applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/pm_static.yml

```diff
@@ -0,0 +1,47 @@
+app:
+  address: 0x8800
+  region: flash_primary
+  size: 0xb8800
+mcuboot:
+  address: 0x0
+  region: flash_primary
+  size: 0x8000
+mcuboot_pad:
+  address: 0x8000
+  region: flash_primary
+  size: 0x800
+mcuboot_primary:
+  address: 0x8000
+  orig_span: &id001
+  - app
+  - mcuboot_pad
+  region: flash_primary
+  size: 0xb9000
+  span: *id001
+mcuboot_primary_app:
+  address: 0x8800
+  orig_span: &id002
+  - app
+  region: flash_primary
+  size: 0xb8800
+  span: *id002
+mcuboot_secondary:
+  address: 0xc1000
+  orig_span: &id003
+  - mcuboot_secondary_pad
+  - mcuboot_secondary_app
+  region: flash_primary
+  size: 0xb9000
+  span: *id003
+mcuboot_secondary_pad:
+  region: flash_primary
+  address: 0xc1000
+  size: 0x800
+mcuboot_secondary_app:
+  region: flash_primary
+  address: 0xc1800
+  size: 0xb8800
+settings_storage:
+  address: 0x17A000
+  region: flash_primary
+  size: 0x3000
```

Comment:
shouldn't we be more generous with settings storage space to improve the lifetime of this memory segment?

Reply:
+1, nRF54L has kinda big memory, and with write-block-size set to 16, the ate now takes more place in the settings partition which also bloats it a bit.

Reply:
I aligned the partition size with boards relying on legacy SoCs (these boards nearly always use `0x2000` or `0x3000` for settings partition size - even on larger SoCs). Still, since nRF54L uses other write-block-size and it has a lot of memory, I updated the settings partition size to `0x5000` here).